app/install/aws-cloudformation.md

```diff
@@ -107,7 +79,7 @@ Provisions Kong resources with new or user provided Postgres DB in a new VPC or
 
 1. **Initial Setup**:
 
-    Create the required key pairs, one to access Kong instances and one for Cassandra if template povisioning a new Cassandra cluster. If you providing your own DB instances, make sure its accessible by Kong instances.
+    Create the required key pair Kong instances. If you providing your own DB instances, make sure its accessible by Kong instances.
```

Comment:
Not sure what the first sentence means, is it `Create the required key pair to access your Kong instances`?
Second sentence:
> If you `are` providing your own DB instances, make sure `it's` accessible `to` Kong instances.